Plots/Panels/NCL_panel_13.py

```diff
@@ -0,0 +1,113 @@
+"""
+NCL_panel_13.py
+===============
+This script illustrates the following concepts:
+   - Overlaying a vector field over filled contours
+   - Paneling two plots vertically
+
+See following URLs to see the reproduced NCL plot & script:
+    - Original NCL script: http://www.ncl.ucar.edu/Applications/Scripts/panel_13.ncl
+    - Original NCL plot: http://www.ncl.ucar.edu/Applications/Images/panel_13_lg.png
+"""
+
+###############################################################################
+# Import packages:
+import cartopy.crs as ccrs
+import matplotlib.pyplot as plt
+from matplotlib.ticker import FormatStrFormatter
+import matplotlib.patches as mpatches
+import numpy as np
+import xarray as xr
+import math
+
+import geocat.datafiles as gdf
+from geocat.viz import cmaps as gvcmaps
+import geocat.viz.util as gvutil
+
+###############################################################################
+# Read in data:
+
+# Open a netCDF data file using xarray default engine and load the data into xarrays
+ds = xr.open_dataset(gdf.get("netcdf_files/uv300.nc"))
+
+# Extract data from second timestep
+ds = ds.isel(time=1).drop_vars('time')
+
+# Ensure longitudes range from 0 to 360 degrees
+U = gvutil.xr_add_cyclic_longitudes(ds.U, "lon")
+V = gvutil.xr_add_cyclic_longitudes(ds.V, "lon")
+
+# Thin data
+U = U[::4, ::4]
+V = V[::4, ::4]
+
+# Calculate the magnitude of the winds
+magnitude = np.sqrt(U.data**2 + V.data**2)
+
+###############################################################################
+# Plot:
+
+# Create sublots and specify their projections
+projection = ccrs.PlateCarree()
+fig, axs = plt.subplots(2, 1, figsize=(7, 10), subplot_kw={"projection": projection})
+plt.tight_layout(pad=4, h_pad=-5)
+
+# Add coastlines
+axs[0].coastlines(linewidth=0.5, zorder=1)
+axs[1].coastlines(linewidth=0.5, zorder=1)
+
+# Use geocat.viz.util convenience function to set axes tick values
+gvutil.set_axes_limits_and_ticks(axs[0], xlim=[-180, 180], ylim=[-90, 90], xticks=np.arange(-180, 181, 30),
+                                 yticks=np.arange(-90, 91, 30))
+gvutil.set_axes_limits_and_ticks(axs[1], xlim=[-180, 180], ylim=[-90, 90], xticks=np.arange(-180, 181, 30),
+                                 yticks=np.arange(-90, 91, 30))
+
+# Use geocat.viz.util convenience function to add minor and major tick lines
+gvutil.add_major_minor_ticks(axs[0])
+gvutil.add_major_minor_ticks(axs[1])
+
+# Use geocat.viz.util convenience function to make plots look like NCL plots by using latitude, longitude tick labels
+gvutil.add_lat_lon_ticklabels(axs[0])
```

Comment:
Remove the degree symbol from each tick mark label to match the NCL plot.

Reply:
It's fixed now